client/apps/edit/components/section_artworks/index.coffee

```diff
@@ -123,10 +148,10 @@ module.exports = React.createClass
           (@props.section.artworks.map (artwork, i) =>
             li { key: i },
               div { className: 'esa-img-container' },
-                img { src: artwork.get('image_urls')?.large }
+                img { src: artwork.get('image_urls')?.large or artwork.attributes.images[0]?.image_urls.large }
```

Comment:
In general you should use `set` and `get` instead of `.attributes` so `artwork.get('images')[0]...`. In this case I don't mind b/c this illuminates a larger refactor I want to do which is drop the orchestration stuff in favor of just v1 data models.